kratos/tests/cpp_tests/containers/test_indirect_scalar_variables.cpp

```diff
@@ -0,0 +1,91 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:		 BSD License
+//                   Kratos default license: kratos/license.txt
```

Comment:
```suggestion
//  License:		 BSD License
//                   Kratos default license: kratos/license.txt
```
Minor in the header formatting.